src/System.IO.FileSystem/src/System/IO/PathHelpers.Unix.cs

```diff
@@ -0,0 +1,27 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.IO
+{
+    internal static partial class PathHelpers
+    {
+        /// <summary>
+        /// Returns fully normalized path.
+        /// </summary>
+        /// <param name="path"></param>
```

Comment:
Please do'nt have <param> tags unless you have content in them. I know VS generates them for you but they're clutter.